tomviz/PipelineView.cxx

```diff
@@ -94,10 +95,13 @@ void PipelineView::contextMenuEvent(QContextMenuEvent* e)
   QMenu contextMenu;
   QAction* cloneAction = nullptr;
   QAction* markAsAction = nullptr;
+  QAction* saveDataAction = nullptr;
   if (dataSource != nullptr) {
     cloneAction = contextMenu.addAction("Clone");
     new CloneDataReaction(cloneAction);
     if (dataSource->type() == DataSource::Volume) {
+      saveDataAction = contextMenu.addAction("Save Data");
+      new SaveDataReaction(saveDataAction);
```

Comment:
You should move it outside the if (dataSource->type() == DataSource::Volume) block.  Save Data applies equally to volumes and tilt series.